ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -68,35 +84,35 @@ Write-Host "Locating '$imageName' image for use in template virtual machine"
 $imageObject = $labPlan | Get-AzLabServicesPlanImage | Where-Object {$_.DisplayName -EQ $imageName -and $_.Sku -EQ $sku -and (-not [string]::IsNullOrEmpty($_.EnabledState))} | Where-Object -Property EnabledState -eq "Enabled"
 
 if($null -eq $imageObject) {
-    Write-Error "Image '$imageName' was not found in the gallery images. No lab was created within lab account $labPlanName."
+    Write-Error "Image '$imageName' was not found in the gallery images. Couldn't create lab $labName."
```

Comment:
We create the lab plan above, I think we're assuming that this particular image "Windows Server 2022 Datacenter (Gen2)" will be enabled by default...  Perhaps we should check and if it's not enabled, enable it automatically?  The user doesn't have an opportunity to jump in to fix it in the middle of the script?

Reply:
Yes, the image "Windows Server 2022 Datacenter (Gen 2)" is enabled automatically.  

We could consider making the code that searches the image a little more broad to future proof it in the future.